src/notification/sound.ts

```diff
@@ -18,29 +19,33 @@ if (config.notifications.playSound) {
 	}
 }
 
-export function playSound() {
```

Comment:
this file is much more readable if you view it w/o whitespace changes: https://github.com/jef/streetmerchant/pull/1411/files?diff=unified&w=1